brands/amenity/clinic.json

```diff
@@ -103,5 +103,25 @@
       "healthcare": "clinic",
       "name": "Terveystalo"
     }
+  },
+  "amenity/clinic|VA Clinic": {
+    "countryCodes": ["us"],
+    "matchNames": [
+      "department of veterans affairs clinic",
+      "veterans administration",
+      "veterans administration clinic",
+      "veterans affairs",
+      "veterans affairs clinic"
+    ],
+    "tags": {
+      "amenity": "clinic",
+      "brand": "VA",
```

Comment:
I would recommend either using the full name or the acronym VHA to prevent confusion between the Department of Veterans Affairs (overall cabinet-level department) and the Veterans Health Administration (subdivision/administration).

Reply:
“Department of Veterans Affairs” and “Veterans Health Administration” only appear on signage etc. the way an `operator` would. But “VA” figures prominently in the logo and the names of many facilities.

Reply:
I just checked some photos and had forgotten how prominent the VA is on signs.